doc/src/stdlib/test.md

```diff
@@ -163,55 +163,6 @@ ERROR: Some tests did not pass: 3 passed, 1 failed, 0 errored, 0 broken.
  in finish at test.jl:362
 ```
 
-## Other Test Macros
-
-As calculations on floating-point values can be imprecise, you can perform approximate equality
-checks using either `@test a ≈ b` (where `≈`, typed via tab completion of `\approx`, is the
-[`isapprox()`](@ref) function) or use [`isapprox()`](@ref) directly.
```

Comment:
This first paragraph seems fine to leave in. Same with the examples that use `isapprox`, and the ` ```@docs ` line for `@inferred`